kuma/landing/templates/landing/promote_buttons.html

```diff
@@ -274,7 +274,7 @@
         // Update the button preview image.
         $('#button-image')
             .attr('alt', sel.alt_text)
-            .attr('src', '{{ MEDIA_URL }}img/promote/promobutton_'+sel.id+'.png');
+            .attr('src', '{{ STATIC_URL }}img/promote/promobutton_'+sel.id+'.png');
```

Comment:
You should be able to do `{{ static('img/promote/promobutton_' + sel.id + '.png'') }}` here since we're using Jinja2 and the limitations of Django's template language doesn't apply here